extension/persistence/eclipselink/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreSessionImpl.java

```diff
@@ -703,7 +703,7 @@ public void deleteFromPolicyMappingRecordsInCurrentTxn(
   @Override
   public void deleteAllEntityPolicyMappingRecordsInCurrentTxn(
       @Nonnull PolarisCallContext callCtx,
-      @Nonnull PolarisEntityCore entity,
+      @Nonnull PolarisBaseEntity entity,
```

Comment:
Not a blocker: I feel like these two classes are interchangeable in Polaris. Can we merge them? cc @dennishuo